state-chain/custom-rpc/src/snapshots/custom_rpc__test__boost_depth_serialization.snap

```diff
@@ -0,0 +1,22 @@
+---
+source: state-chain/custom-rpc/src/lib.rs
+expression: val
+---
+[
+  {
+    "asset": {
+      "chain": "Ethereum",
+      "asset": "FLIP"
+    },
+    "tier": 10,
+    "available_amount": 1000000000000000000000000000000000000
```

Comment:
Should we not serialize amounts using hex? ( @GabrielBuragev ?)

Reply:
Yeah, was thinking about that too as I was writing this test. Don't think JS will be able to read this number accurately. I thought if the PRs are merged quickly, I'd make the change in a separate PR (since the PR for the other RPC method is already merged anyway), but since there are some other comments to address anyway, I will update it to use the hex amounts too.